src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -440,6 +440,15 @@ private function createFirewall(ContainerBuilder $container, string $id, array $
         // Determine default entry point
         $configuredEntryPoint = $firewall['entry_point'] ?? null;
 
+        // When a entry point path is configured, it overrides the default entry point
```

Comment:
What if one set both `entry_point` and `entry_point_path` at the firewall level? Should we forbid such mix for clarity? A test would be nice for that case btw

Reply:
Currently, if both options are set, the entry point path overrides the entry point.
It could be a good thing to throw an exception if both are set to avoid confusion and ease debug.
I try to add a test if I had the time.